doc/sphinx/addendum/omega.rst

```diff
@@ -5,6 +5,19 @@ Omega: a solver for quantifier-free problems in Presburger Arithmetic
 
 :Author: Pierre Crégut
 
+.. warning::
+
+   The :tacn:`omega` tactic is about to be deprecated in favor of the
```

Comment:
"is about" -> "is scheduled to be deprecated in favor of the `lia` tactic in the next Coq release" ?

Reply:
I can do this change, but only if @coq/micromega-maintainers actually confirm that this is going to happen. Otherwise, I'd rather remain vague.

Reply:
If @ejgallego commits to making it happen, why not. But otherwise, please don't announce that we scheduled something when we did not.